src/conventions.md

```diff
@@ -0,0 +1,59 @@
+# Notational conventions
+
+## Conventions
+
+Like all technical books, this book has certain conventions in how it displays information. These conventions are documented here.
+
+* Statements that define a term contain that term in *italics*. Whenever that term is used outside of that chapter, it is usually a link to the section that has this definition.
+
+  An *example term* is an example of a term being defined.
+
+* The main text describes the latest stable edition. Differences to previous editions are separated in edition blocks:
+
+  > [!EDITION-2018]
+  > Before the 2018 edition, the behavior was this. As of the 2018 edition, the behavior is that.
+
+* Notes that contain useful information about the state of the book or point out useful, but mostly out of scope, information are in note blocks.
+
+  > [!NOTE]
+  > This is an example note.
+
+* Example blocks show an example that demonstrates some rule or points out some interesting aspect. Some examples may have hidden lines which can be viewed by clicking the eye icon that appears when hovering or tapping the example.
+
+  > [!EXAMPLE]
+  > This is a code example.
+  > ```rust
+  > println!("hello world");
+  > ```
+
+* Warnings that show unsound behavior in the language or possibly confusing interactions of language features are in a special warning box.
+
+  > [!WARNING]
+  > This is an example warning.
+
+* Code snippets inline in the text are inside `<code>` tags.
+
+  Longer code examples are in a syntax highlighted box that has controls for copying, executing, and showing hidden lines in the top right corner.
+
+  ```rust
+  # // This is a hidden line.
+  fn main() {
+      println!("This is a code example");
+  }
+  ```
+
+  All examples are written for the latest edition unless otherwise stated.
+
+* The grammar and lexical productions are described in the [Notation] chapter.
+
+r[example.rule.label]
```

Comment:
shows that the notation accepts dashes
`[general.less-general.less-specific.specific]`

anyways, real reason I propose this is that seeing "example" here made me think of code example, perhaps because I just how a code example is done (in the previous bullet point)